### DISCUSSION TITLE
v1.0

### DISCUSSION THREAD

**Discussion by scriptingosx** [jamf/Setup-Manager#3] [Announcements]
- updated to new Jamf Concepts Use Agreement
- updated German and Swedish localizations
- added name for macOS 15
- new `hideActionLabels` and `hideDebugLabel` keys
- 'Jamf ID' is now only visible in the extended 'About this Mac' View (reachable when holding the option key)
- messaging when Setup Manager is launched in user space or with missing configuration
- UI tweaks

<hr /><em>This discussion was created from the release <a href='https://github.com/Jamf-Concepts/Setup-Manager/releases/tag/v1.0'>v1.0</a>.</em>

**Comment by dan-snelson** [comment]
Congratulations! One word:

> Sweetness
